tasks/agent-linux.yml

```diff
@@ -94,7 +94,7 @@
     name: datadog-agent-sysprobe
     state: stopped
     enabled: no
-  when: not datadog_skip_running_check and (not datadog_enabled or not datadog_sysprobe_enabled)
+  when: not datadog_skip_running_check and (not datadog_enabled or not datadog_sysprobe_enabled) and ansible_facts.services['datadog-agent-sysprobe'] is defined
```

Comment:
We'll want to check both type of services as done here: https://github.com/DataDog/ansible-datadog/blob/8db7a6a88cf40435bb78e54a22dabe8d4fd23b47/tasks/agent-linux.yml#L83

Perhaps we can create a fact about whether the service is installed or not (`datadog_sysprobe_installed`?) and use that here and in the `datadog_sysprobe_enabled` fact check.

Reply:
Yeah good shout, I'll sort that

Reply:
https://github.com/DataDog/ansible-datadog/pull/255/commits/5299d1ae4bf2af14474d83d4d19ab33a1e613df9